packages/components/src/radio-control/types.ts

```diff
@@ -24,6 +24,10 @@ export type RadioControlProps = Pick<
 		 * The internal value compared against select and passed to onChange
 		 */
 		value: string;
+		/**
+		 * Optional help text to be shown in addition the label.
+		 */
+		helpText?: string;
```

Comment:
I'm torn on whether we should do this, or make it `help?: ReactNode` to have consistency with the BaseControl family APIs ☹️

Reply:
Making it a `ReactNode` would align with `BaseControl` and would also allow for [breaking help text into separate paragraphs](https://github.com/WordPress/gutenberg/pull/63751#discussion_r1688124771).

I'm just hesitant to allow consumers to pass any markup as the help text — but I guess really determined consumers would find a way anyway, like they already do to pass the help text without an official API.

If we go ahead with the change, should we rename the property to `help`, since users can pass any markup (and not just text) ?

Reply:
Update: breaking text into separate paragraphs is actually something that [we should discourage](https://github.com/WordPress/gutenberg/pull/63751/files#r1688327043).

The safest approach to me seems to be:
- to rename `helpText` to `help` (more generic and future-proof)
- to keep the type as `string` (it can always be widened to `ReactNode` later, if needed

Reply:
> If we go ahead with the change, should we rename the property to `help`, since users can pass any markup (and not just text) ?

Yes, I think so. And we could prevent some misuse by documenting the best practices, like in BaseControl itself:

https://github.com/WordPress/gutenberg/blob/495ad8c5fe99b43878b854a9e97d3b477acff885/packages/components/src/base-control/types.ts#L24

We could also add a sentence about it being intended for descriptions that can be experienced as plain text ([from MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby#description)):

> The `aria-describedby` property is appropriate when the associated content contains descriptions experienced as plain text.

Reply:
Ok yes, I think that makes sense. Let's go with `description?: string`.